### PR TITLE
Upgrade to v2

### PR DESCRIPTION
**Pull request by rlindskog**
Addresses #13 

Deployed example - https://graphql-pokemon2.vercel.app

Run locally with `vercel dev` to test.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
   "repository": "https://github.com/lucasbento/graphql-pokemon",
   "scripts": {
     "eslint": "eslint src/*",
-    "build": "rm -rf dist/* && babel src --out-dir dist --copy-files",
+    "build-app": "rm -rf dist/* && babel src --out-dir dist --copy-files",
```
Comment (2XL) [inline package.json:44]: this change is not very clear

File: README.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ query {
 
 ```sh
 yarn
-yarn run build
+yarn run build-app
```
Comment (2XL) [inline README.md:58]: here as well

File: README.md [hunk 3]
```diff
@@ -64,6 +64,10 @@ yarn start
 ```sh
 yarn
 yarn run watch # Using nodemon for auto-reloading
+
+or
+
+vercel dev # npm i -g @now/cli
```
Comment (leerob) [inline README.md:70]:
```suggestion
vercel dev # npm i -g vercel
```

**Comment by leerob** [comment]
I'd love to see this merged! Was looking for a Pokemon GQL API today 😄

**Comment by kfnx** [comment]
Which part of the changes that update to use the pokeAPI to work with v2?

**Comment by carloslibardo** [comment]
Why don't be merged yet? For me it's right ;)

**Comment by leerob** [comment]
Hey! Any update here? We're seen some issues from folks referencing this URL as it's now broken.

If this project is deprecated, let me know and we can reassign the URL 👍

**Comment by lucasbento** [comment]
@leerob: merged, thank you so much for doing this and I'm very very sorry for not looking into it sooner.

**Pull request merged**